tariff/tariff.go

```diff
@@ -0,0 +1,68 @@
+package tariff
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/provider"
+	"github.com/evcc-io/evcc/util"
+	"github.com/jinzhu/now"
+)
+
+type Tariff struct {
```

Comment:
Vorschlag: Custom statt Tariff
```suggestion
type Custom struct {
```

Tariff klingt sehr allgemein. Das hier ist ja schon eher ein Randfall.

Reply:
Ist aber in line mit Meter/Charger etc. Daher wontfix!

Reply:
Dann lieber mit separatem PR für alle umstellen.

Reply:
Ok